src/bliss/bench_pgm.h

```diff
@@ -0,0 +1,39 @@
+#ifndef BLISS_BENCH_PGM
+#define BLISS_BENCH_PGM
+
+#include <vector>
+
+#include "bliss/bliss_index.h"
+#include "include/pgm/pgm_index_dynamic.hpp"
+
+
+namespace bliss {
+
+template <typename KEY_TYPE, typename VALUE_TYPE>
+class BlissPGMIndex : public BlissIndex<KEY_TYPE, VALUE_TYPE> {
+   public:
+    // pgm::PGMIndex<int, epsilon> _index();
```

Comment:
can we get rid of unused lines of code that have been commented?

Reply:
Got rid of it and added sanity checks!